cmd/locate.go

```diff
@@ -37,17 +36,42 @@ func init() {
 }
 
 func Locate(
-	buildlistFile string,
 	locatedFile string,
 ) error {
-	var buildlist []gotool.Module
-	err := ReadJSONFile(buildlistFile, &buildlist)
+	tidyCmd := exec.Command("go", "mod", "tidy")
```

Comment:
make sure our dependencies are up-to-date. Note that we assume this command is being run in the top level directory of a go project. Is that ok or should we leave tidy'ing and vendor'ing up to the user?